### PR TITLE
Add make-release.sh script to automate the release process

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add make-release.sh script that automates certain steps of release process

### CODE REVIEW

File: make-release.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+git checkout "${BASEBRANCH}"
+
+# create new branch off ${BASEBRANCH} (or check out latest commits if branch already exists), then push to origin
+if [[ "${BASEBRANCH}" != "${BRANCH}" ]]; then
+  git branch "${BRANCH}" || git checkout "${BRANCH}" && git pull origin "${BRANCH}"
+  git push origin "${BRANCH}"
+  git fetch origin "${BRANCH}:${BRANCH}"
+  git checkout "${BRANCH}"
+fi
+
```
Comment (nickboldt) [inline make-release.sh]: see comment on https://github.com/eclipse/che-docs/pull/1087/files#r380674902

File: make-release.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    lastCommitComment="$(git log -1 --pretty=%B)"
+    hub pull-request -o -f -m "${lastCommitComment}
+${lastCommitComment}" -b "${BRANCH}" -h "${PR_BRANCH}"
+  fi 
+fi
+
+popd > /dev/null || exit
+
+# cleanup tmp dir
+cd /tmp && rm -fr "$TMP"
```
Comment (nickboldt) [inline make-release.sh]: see comment on https://github.com/eclipse/che-docs/pull/1087/files#r380673958

File: make-release.sh [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+fi
+
+if [[ $TRIGGER_RELEASE -eq 1 ]]; then
+  # push new branch to release branch to trigger CI build
+  git fetch origin "${BRANCH}:${BRANCH}"
+  git checkout "${BRANCH}"
+  git branch release -f 
+  git push origin release -f
+fi
+
```
Comment (nickboldt) [inline make-release.sh]: Need to bump 7.9.x branch to 7.9.1-SNAPSHOT here to prepare that branch for future work.

eg., `mvn versions:set -DgenerateBackupPoms=false -DnewVersion=$1"` etc.

Comment (mkuznyetsov) [reply make-release.sh]: resolved

File: make-release.sh (hunk 2 above)
Comment (mkuznyetsov) [reply make-release.sh]: resolved

**Pull request merged**